api/swagger.yaml

```diff
@@ -0,0 +1,788 @@
+openapi: 3.0.1
+info:
+  title: Space Tycoon
+  description: Space Tycoon server.
+  contact:
+    url: https://www.garage-trip.cz/
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more about Space Tycoon game
+  url: https://github.com/gdg-garage/space-tycoon
+servers:
+- url: https://space-tycoon.garage-trip.cz/
+tags:
+- name: login
+  description: Get user session
+- name: staticdata
+  description: Static game state (planets)
+- name: data
+  description: "Dynamic game data (scores, prices, spaceship positions)"
+- name: commands
+  description: Specify commands for your ships
+- name: tick
+  description: Signal that your turn is over for the current tick
+paths:
+  /login:
+    post:
+      tags:
+      - login
+      requestBody:
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/Credentials'
```

Comment:
we should us id everywhere.
(planets and ships do not have unique names as they can be changed by players. therefore it would make sense to use id instead of names when targeting planets and ships and therefore everywhere else too)

Reply:
I agree, we reference players with IDs too in the rest of the api

Reply:
On the other hand, for login, it makes prolly more sense to use name than ID (and in the game, we reference with player id, not user id)

Reply:
user: string
password: string
player: id